packages/components/src/InputRange/_docs/InputRange.mdx

```diff
@@ -0,0 +1,36 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as InputRangeStories from "./InputRange.stories"
+
+<Meta of={InputRangeStories} />
+
+# InputRange
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/InputRange"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { InputRange } from "@kaizen/components"'
+/>
+
+## Overview
+
+A range of things.
```

Comment:
Should we describe what kind of thing? Or is it totally open to the consumer

Reply:
Up to them!